pontoon/base/static/js/main.js

```diff
@@ -156,6 +172,9 @@ var Pontoon = (function (my) {
             '</header>' +
             '<p class="original">' + self.doNotRender(data.original || '') + '</p>' +
             '<p class="translation" dir="auto" lang="' + self.locale.code + '">' +
+              self.markPlaceables(self.doNotRender(data.translation)) +
```

Comment:
Or even better -> use self.doNotRender inside MarkPlaceables to avoid code duplication.